### DISCUSSION TITLE
使用DELETE语句删除表记录，实际的分区和磁盘数据没有删除

### DISCUSSION THREAD

**Discussion by TheThirdReich77** [apache/doris#35772] [A - General / Q&A]
doris版本：2.0.9
表模型和分区：duplicate key、按st_date_part时间列进行按日分区
删除语句：delete from events_raw where st_date_part < '2024-05-14'

删除后该时间点之前的行记录已经不可见，通过show delete语句查询到事务已是finished状态，但过了3天后记录并没有删除，分区数据仍然存在，磁盘空间未清除
![WX20240603-101921](https://github.com/apache/doris/assets/44658772/a9d2ea32-3eef-4f17-afdc-b42e72bbdbe4)

![WX20240603-102217](https://github.com/apache/doris/assets/44658772/12fca0cb-ac81-4e6a-88c3-ecc1d656cab6)

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题

**Reply by laoba25** [reply]
为什么每个下边都会看到这个回复？ selectdb 和 Apache doris 是什么关系呢？
